#! /bin/sh /usr/share/dpatch/dpatch-run
## 12_network_images.dpatch by Roland Rosenfeld <roland@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Some network icons colored and/or combined by Roland Rosenfeld.

@DPATCH@
diff -urNad xfig~/Libraries/Networks/Devices/firewall-pair.fig xfig/Libraries/Networks/Devices/firewall-pair.fig
--- Libraries/Networks/Devices/firewall-pair.fig
+++ Libraries/Networks/Devices/firewall-pair.fig
@@ -0,0 +1,75 @@
+#FIG 3.2  Produced by xfig version 3.2.5-alpha5
+Portrait
+Center
+Inches
+A4      
+100.00
+Single
+-2
+# A pair of firewalls
+# Combined by Roland Rosenfeld
+1200 2
+0 32 #aeaaae
+# A pair of firewalls
+6 3375 2175 4050 3075
+6 3600 2250 4050 3075
+# Firewall
+# (from TomiOllila <Tomi.Ollila@tfi.net> )
+6 3600 2250 4050 3075
+2 1 0 1 0 7 0 0 -1 0.000 0 0 -1 0 0 2
+	 3825 3075 3825 2475
+2 1 0 1 0 20 2 0 20 0.000 0 0 -1 0 0 5
+	 3675 2475 3675 3075 3825 3075 3975 2925 3975 2400
+2 1 0 1 0 32 1 0 20 0.000 0 0 -1 0 0 7
+	 3600 2400 3600 2475 3900 2475 4050 2325 4050 2250 3750 2250
+	 3600 2400
+2 1 0 1 0 7 0 0 -1 0.000 0 0 -1 0 0 3
+	 3900 2475 3900 2400 3600 2400
+2 1 0 1 0 7 0 0 -1 0.000 0 0 -1 0 0 2
+	 3900 2400 4050 2250
+2 1 0 1 0 7 0 0 -1 0.000 0 0 -1 0 0 3
+	 3675 2625 3825 2625 3975 2475
+2 1 0 1 0 7 0 0 -1 0.000 0 0 -1 0 0 3
+	 3675 2775 3825 2775 3975 2625
+2 1 0 1 0 7 0 0 -1 0.000 0 0 -1 0 0 3
+	 3675 2925 3825 2925 3975 2775
+2 1 0 1 0 7 0 0 -1 0.000 0 0 -1 0 0 2
+	 3900 3000 3900 2850
+2 1 0 1 0 7 0 0 -1 0.000 0 0 -1 0 0 2
+	 3900 2700 3900 2550
+2 1 0 1 0 7 0 0 -1 0.000 0 0 -1 0 0 2
+	 3750 2475 3750 2625
+2 1 0 1 0 7 0 0 -1 0.000 0 0 -1 0 0 2
+	 3750 2775 3750 2925
+-6
+-6
+6 3375 2175 3825 3000
+6 3450 2175 3825 3000
+2 1 0 1 0 7 20 0 -1 0.000 0 0 7 0 0 2
+	 3600 3000 3600 2400
+2 1 0 1 0 7 20 0 -1 0.000 0 0 7 0 0 2
+	 3675 2325 3825 2175
+2 1 0 1 0 7 20 0 -1 0.000 0 0 7 0 0 3
+	 3450 2550 3600 2550 3750 2400
+2 1 0 1 0 7 20 0 -1 0.000 0 0 7 0 0 3
+	 3450 2700 3600 2700 3750 2550
+2 1 0 1 0 7 20 0 -1 0.000 0 0 7 0 0 3
+	 3450 2850 3600 2850 3750 2700
+2 1 0 1 0 7 20 0 -1 0.000 0 0 7 0 0 2
+	 3675 2925 3675 2775
+2 1 0 1 0 7 20 0 -1 0.000 0 0 7 0 0 2
+	 3675 2625 3675 2475
+2 1 0 1 0 7 20 0 -1 0.000 0 0 7 0 0 2
+	 3525 2400 3525 2550
+2 1 0 1 0 7 20 0 -1 0.000 0 0 7 0 0 2
+	 3525 2700 3525 2850
+-6
+2 1 0 1 0 20 22 0 20 0.000 0 0 -1 0 0 5
+	 3450 2400 3450 3000 3600 3000 3750 2850 3750 2325
+2 1 0 1 0 32 21 0 20 0.000 0 0 -1 0 0 7
+	 3375 2325 3375 2400 3675 2400 3825 2250 3825 2175 3525 2175
+	 3375 2325
+2 1 0 1 0 7 20 0 -1 0.000 0 0 -1 0 0 3
+	 3675 2400 3675 2325 3375 2325
+-6
+-6
diff -urNad xfig~/Libraries/Networks/Devices/internet.fig xfig/Libraries/Networks/Devices/internet.fig
--- Libraries/Networks/Devices/internet.fig
+++ Libraries/Networks/Devices/internet.fig
@@ -0,0 +1,39 @@
+#FIG 3.2  Produced by xfig version 3.2.5-alpha5
+Portrait
+Center
+Inches
+A4      
+100.00
+Single
+-2
+# Internet cloud
+# By Roland Rosenfeld
+1200 2
+# An Internet cloud
+6 4275 5700 7200 7650
+2 1 0 0 7 7 970 0 20 4.000 0 0 0 0 0 65
+	 4719 7111 4633 7121 4548 7105 4469 7057 4394 6983 4346 6888
+	 4309 6776 4298 6654 4309 6526 4346 6415 4394 6325 4469 6244
+	 4548 6197 4633 6181 4719 6197 4798 6048 4893 5915 4994 5825
+	 5122 5746 5244 5719 5377 5719 5505 5746 5627 5825 5733 5915
+	 5850 5825 5961 5746 6094 5719 6222 5719 6355 5746 6472 5825
+	 6583 5915 6668 6048 6748 6197 6833 6181 6918 6197 7003 6244
+	 7072 6325 7130 6415 7157 6526 7168 6654 7157 6776 7130 6888
+	 7072 6983 7003 7057 6918 7105 6833 7121 6748 7111 6668 7265
+	 6583 7387 6472 7488 6355 7557 6222 7594 6094 7594 5961 7557
+	 5850 7488 5733 7387 5627 7488 5505 7557 5377 7594 5244 7594
+	 5122 7557 4994 7488 4893 7387 4798 7265 4719 7111
+2 1 0 2 1 0 969 0 -1 0.000 0 0 0 0 0 65
+	 4719 7111 4633 7121 4548 7105 4469 7057 4394 6983 4346 6888
+	 4309 6776 4298 6654 4309 6526 4346 6415 4394 6325 4469 6244
+	 4548 6197 4633 6181 4719 6197 4798 6048 4893 5915 4994 5825
+	 5122 5746 5244 5719 5377 5719 5505 5746 5627 5825 5733 5915
+	 5850 5825 5961 5746 6094 5719 6222 5719 6355 5746 6472 5825
+	 6583 5915 6668 6048 6748 6197 6833 6181 6918 6197 7003 6244
+	 7072 6325 7130 6415 7157 6526 7168 6654 7157 6776 7130 6888
+	 7072 6983 7003 7057 6918 7105 6833 7121 6748 7111 6668 7265
+	 6583 7387 6472 7488 6355 7557 6222 7594 6094 7594 5961 7557
+	 5850 7488 5733 7387 5627 7488 5505 7557 5377 7594 5244 7594
+	 5122 7557 4994 7488 4893 7387 4798 7265 4719 7111
+4 0 0 100 0 18 33 0.0000 4 405 2040 4744 6853 Internet\001
+-6
diff -urNad xfig~/Libraries/Networks/Routers/router-pair.fig xfig/Libraries/Networks/Routers/router-pair.fig
--- Libraries/Networks/Routers/router-pair.fig
+++ Libraries/Networks/Routers/router-pair.fig
@@ -0,0 +1,78 @@
+#FIG 3.2  Produced by xfig version 3.2.5-alpha5
+Portrait
+Center
+Inches
+A4      
+100.00
+Single
+-2
+# A pair of routers
+# Colored and combined by Roland Rosenfeld
+1200 2
+# A pair of routers
+6 3150 3300 4275 3975
+6 3346 3450 4269 3975
+6 3844 3615 4173 3718
+# polyline
+2 1 0 1 0 7 50 0 20 0.000 0 0 0 0 0 8
+	 3879 3701 3854 3647 3976 3624 3955 3642 4159 3676 4111 3713
+	 3908 3676 3879 3701
+-6
+6 3617 3636 3803 3759
+# polyline
+2 1 0 1 0 7 50 0 20 0.000 0 0 0 0 0 8
+	 3764 3731 3638 3754 3634 3694 3671 3703 3738 3646 3800 3657
+	 3727 3719 3764 3731
+-6
+6 3803 3470 3988 3594
+# polyline
+2 1 0 1 0 7 50 0 20 0.000 0 0 7 0 0 8
+	 3837 3518 3982 3481 3983 3541 3947 3535 3876 3591 3809 3581
+	 3882 3526 3837 3518
+-6
+1 2 0 1 0 15 51 0 20 0.000 1 0.0000 3807 3619 457 169 3349 3619 4264 3619
+1 2 0 1 0 15 53 0 20 0.000 1 0.0000 3803 3806 457 169 3346 3806 4261 3806
+# polyline
+2 1 0 1 0 7 50 0 20 0.000 0 0 0 0 0 8
+	 3705 3557 3738 3608 3609 3637 3638 3615 3439 3581 3488 3545
+	 3681 3576 3705 3557
+2 1 0 1 0 7 50 -1 -1 0.000 0 0 -1 0 0 2
+	 3350 3613 3347 3812
+2 1 0 1 0 7 50 -1 -1 0.000 0 0 -1 0 0 2
+	 4263 3626 4263 3815
+2 2 0 0 0 15 52 -1 20 0.000 0 0 -1 0 0 5
+	 3349 3626 4269 3626 4269 3800 3349 3800 3349 3626
+-6
+6 3150 3300 4125 3825
+6 3600 3450 3975 3600
+# polyline
+2 1 0 1 0 7 60 0 20 0.000 0 0 0 0 0 8
+	 3683 3551 3658 3497 3780 3474 3759 3492 3963 3526 3915 3563
+	 3712 3526 3683 3551
+-6
+6 3600 3300 3825 3450
+# polyline
+2 1 0 1 0 7 60 0 20 0.000 0 0 7 0 0 8
+	 3641 3368 3786 3331 3787 3391 3751 3385 3680 3441 3613 3431
+	 3686 3376 3641 3368
+-6
+6 3375 3450 3675 3675
+# polyline
+2 1 0 1 0 7 60 0 20 0.000 0 0 0 0 0 8
+	 3568 3581 3442 3604 3438 3544 3475 3553 3542 3496 3604 3507
+	 3531 3569 3568 3581
+-6
+1 2 0 1 0 15 61 0 20 0.000 1 0.0000 3611 3469 457 169 3153 3469 4068 3469
+1 2 0 1 0 15 63 0 20 0.000 1 0.0000 3607 3656 457 169 3150 3656 4065 3656
+2 1 0 1 0 7 60 -1 -1 0.000 0 0 -1 0 0 2
+	 4067 3476 4067 3665
+# polyline
+2 1 0 1 0 7 60 0 20 0.000 0 0 0 0 0 8
+	 3509 3407 3542 3458 3413 3487 3442 3465 3243 3431 3292 3395
+	 3485 3426 3509 3407
+2 1 0 1 0 7 60 -1 -1 0.000 0 0 -1 0 0 2
+	 3154 3463 3151 3662
+2 2 0 0 0 15 62 -1 20 0.000 0 0 -1 0 0 5
+	 3153 3476 4073 3476 4073 3650 3153 3650 3153 3476
+-6
+-6
diff -urNad xfig~/Libraries/Networks/Routers/router4.fig xfig/Libraries/Networks/Routers/router4.fig
--- Libraries/Networks/Routers/router4.fig
+++ Libraries/Networks/Routers/router4.fig
@@ -0,0 +1,44 @@
+#FIG 3.2  Produced by xfig version 3.2.5-alpha5
+Portrait
+Center
+Metric
+A4      
+100.00
+Single
+-2
+# Router
+# Colored by Roland Rosenfeld
+1200 2
+# Router
+6 3870 4455 5940 5625
+6 4995 4815 5715 5040
+# polyline
+2 1 0 1 0 7 50 0 20 0.000 0 0 0 0 0 8
+	 5072 5002 5018 4886 5284 4834 5238 4873 5683 4948 5578 5030
+	 5136 4948 5072 5002
+-6
+6 4500 4860 4905 5130
+# polyline
+2 1 0 1 0 7 50 0 20 0.000 0 0 0 0 0 8
+	 4822 5067 4545 5117 4537 4989 4618 5007 4766 4883 4900 4905
+	 4741 5041 4822 5067
+-6
+6 4905 4500 5310 4770
+# polyline
+2 1 0 1 0 7 50 0 20 0.000 0 0 7 0 0 8
+	 4981 4603 5296 4523 5300 4653 5222 4640 5066 4762 4919 4741
+	 5079 4622 4981 4603
+-6
+1 2 0 1 0 15 51 0 20 0.000 1 0.0000 4914 4824 999 369 3915 4824 5913 4824
+1 2 0 1 0 15 53 0 20 0.000 1 0.0000 4908 5231 999 369 3909 5231 5907 5231
+# polyline
+2 1 0 1 0 7 50 0 20 0.000 0 0 0 0 0 8
+	 4693 4690 4766 4800 4483 4863 4545 4814 4111 4740 4219 4661
+	 4640 4731 4693 4690
+2 1 0 1 0 7 50 -1 -1 0.000 0 0 -1 0 0 2
+	 3917 4811 3910 5244
+2 1 0 1 0 7 50 -1 -1 0.000 0 0 -1 0 0 2
+	 5910 4838 5910 5251
+2 2 0 0 0 15 52 -1 20 0.000 0 0 -1 0 0 5
+	 3915 4838 5924 4838 5924 5218 3915 5218 3915 4838
+-6
diff -urNad xfig~/Libraries/Networks/Switches/switch-pair.fig xfig/Libraries/Networks/Switches/switch-pair.fig
--- Libraries/Networks/Switches/switch-pair.fig
+++ Libraries/Networks/Switches/switch-pair.fig
@@ -0,0 +1,66 @@
+#FIG 3.2  Produced by xfig version 3.2.5-alpha5
+Portrait
+Center
+Inches
+A4      
+100.00
+Single
+-2
+# A pair of switches
+# Colored and combined by Roland Rosenfeld
+1200 2
+# A pair of switches
+6 2550 2625 4725 3750
+6 2625 2850 4725 3750
+6 3000 2850 4425 3300
+# polyline
+2 1 0 1 0 7 50 0 20 0.000 0 0 0 0 0 8
+	 3603 3180 3581 3210 3194 3210 3161 3240 3028 3195 3268 3126
+	 3214 3184 3608 3184
+# polyline
+2 1 0 1 0 7 50 0 20 0.000 0 0 0 0 0 8
+	 3651 3128 3673 3098 4060 3098 4093 3068 4226 3113 3986 3182
+	 4040 3124 3646 3124
+# polyline
+2 1 0 1 0 7 50 0 20 0.000 0 0 0 0 0 8
+	 3760 3008 3738 3038 3351 3038 3318 3068 3185 3023 3425 2954
+	 3371 3012 3765 3012
+# polyline
+2 1 0 1 0 7 50 0 20 0.000 0 0 0 0 0 8
+	 3827 2963 3849 2933 4236 2933 4269 2903 4402 2948 4162 3017
+	 4216 2959 3822 2959
+-6
+2 1 0 1 0 15 51 0 20 0.000 0 0 -1 0 0 4
+	 4125 3300 4725 2850 4725 3300 4125 3750
+2 2 0 1 0 15 51 0 20 0.000 0 0 -1 0 0 5
+	 2625 3300 4125 3300 4125 3750 2625 3750 2625 3300
+2 3 0 1 0 15 51 -1 20 0.000 0 0 -1 0 0 5
+	 2625 3300 3225 2850 4725 2850 4125 3300 2625 3300
+-6
+6 2550 2625 4650 3525
+6 2925 2625 4350 3075
+# polyline
+2 1 0 1 0 7 61 0 20 0.000 0 0 0 0 0 8
+	 3528 2955 3506 2985 3119 2985 3086 3015 2953 2970 3193 2901
+	 3139 2959 3533 2959
+# polyline
+2 1 0 1 0 7 61 0 20 0.000 0 0 0 0 0 8
+	 3576 2903 3598 2873 3985 2873 4018 2843 4151 2888 3911 2957
+	 3965 2899 3571 2899
+# polyline
+2 1 0 1 0 7 61 0 20 0.000 0 0 0 0 0 8
+	 3685 2783 3663 2813 3276 2813 3243 2843 3110 2798 3350 2729
+	 3296 2787 3690 2787
+# polyline
+2 1 0 1 0 7 61 0 20 0.000 0 0 0 0 0 8
+	 3752 2738 3774 2708 4161 2708 4194 2678 4327 2723 4087 2792
+	 4141 2734 3747 2734
+-6
+2 1 0 1 0 15 62 0 20 0.000 0 0 -1 0 0 4
+	 4050 3075 4650 2625 4650 3075 4050 3525
+2 2 0 1 0 15 62 0 20 0.000 0 0 -1 0 0 5
+	 2550 3075 4050 3075 4050 3525 2550 3525 2550 3075
+2 3 0 1 0 15 62 -1 20 0.000 0 0 -1 0 0 5
+	 2550 3075 3150 2625 4650 2625 4050 3075 2550 3075
+-6
+-6
diff -urNad xfig~/Libraries/Networks/Switches/switch2.fig xfig/Libraries/Networks/Switches/switch2.fig
--- Libraries/Networks/Switches/switch2.fig
+++ Libraries/Networks/Switches/switch2.fig
@@ -0,0 +1,38 @@
+#FIG 3.2  Produced by xfig version 3.2.5-alpha5
+Portrait
+Center
+Inches
+A4      
+100.00
+Single
+-2
+# Switch
+# colored by Roland Rosenfeld
+1200 2
+# Switch
+6 2625 2850 4725 3750
+6 3000 2850 4425 3300
+# polyline
+2 1 0 1 0 7 50 0 20 0.000 0 0 0 0 0 8
+	 3603 3180 3581 3210 3194 3210 3161 3240 3028 3195 3268 3126
+	 3214 3184 3608 3184
+# polyline
+2 1 0 1 0 7 50 0 20 0.000 0 0 0 0 0 8
+	 3651 3128 3673 3098 4060 3098 4093 3068 4226 3113 3986 3182
+	 4040 3124 3646 3124
+# polyline
+2 1 0 1 0 7 50 0 20 0.000 0 0 0 0 0 8
+	 3760 3008 3738 3038 3351 3038 3318 3068 3185 3023 3425 2954
+	 3371 3012 3765 3012
+# polyline
+2 1 0 1 0 7 50 0 20 0.000 0 0 0 0 0 8
+	 3827 2963 3849 2933 4236 2933 4269 2903 4402 2948 4162 3017
+	 4216 2959 3822 2959
+-6
+2 1 0 1 0 15 51 0 20 0.000 0 0 -1 0 0 4
+	 4125 3300 4725 2850 4725 3300 4125 3750
+2 2 0 1 0 15 51 0 20 0.000 0 0 -1 0 0 5
+	 2625 3300 4125 3300 4125 3750 2625 3750 2625 3300
+2 3 0 1 0 15 51 -1 20 0.000 0 0 -1 0 0 5
+	 2625 3300 3225 2850 4725 2850 4125 3300 2625 3300
+-6
